lib/code_climate/test_reporter/formatter.rb

```diff
@@ -15,9 +15,11 @@ class Formatter
       def format(result)
         return true unless CodeClimate::TestReporter.run?
 
-        print "Coverage = #{result.source_files.covered_percent.round(2)}%. "
+        simplecov_result = SimpleCov::Result.new(result.first[1]["coverage"])
```

Comment:
Ideally I think we should have a nice error for an input file we can't handle. I think I would `raise` our own error here if there's no `1` index, no `"coverage"` key, or `SimpleCov::Result.new` itself raises. WDYT?